build-system/eslint-rules/is-experiment-on.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+module.exports = function(context) {
+  const isExperimentOn = 'CallExpression[callee.name=isExperimentOn]';
+  const message = 'isExperimentOn must be passed an explicit string';
```

Comment:
The only OK usages are `AMP.isExperimentOn` and `isExperimentOn`'s own tests, because the caller themselves will have a `'amp-experiment'` string passed to it (so it's already grepable).